src/cohort-manager.js

```diff
@@ -6,10 +6,14 @@ class CohortManager {
   }
 
   createCohort(name) {
+    if (name.length === 0) throw new Error('Cohort needs a name!')
```

Comment:
not sure if it is an issue, but running createCohort() with no parameters at all throws this: 

TypeError: Cannot read properties of undefined (reading 'length')     
at CohortManager.createCohort (file:///C:/Users/chloe/Boolean2023/tdd-cohort-manager-challenge/src/cohort-manager.js:12:14)

meaning that your guard clause itself does not trigger, seeing as it is the condition itself that cannot be read.

Reply:
Ill fix this with a !name. Thanks for spotting this.